test/basic.js

```diff
@@ -407,7 +407,10 @@ function createIgnoreOutDirTest (opts, distPath) {
       },
       function (cb) {
         // create file to ensure that directory will be not ignored because empty
-        fs.open(path.join(outDir, 'ignoreMe'), 'w', cb)
+        fs.open(path.join(outDir, 'ignoreMe'), 'w', function (err, fd) {
+          if (err) return cb(err)
+          fs.close(fd, cb)
+        })
```

Comment:
Given the way that Windows file locks work, I would be surprised if anyone tried running the tests on Windows lately and had them all pass.